src/main/scala/com.snowplowanalytics/forex/ZonedClock.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2015-2019 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.forex
+
+import java.time.ZonedDateTime
+
+import cats.Eval
+import cats.effect.Sync
+
+trait ZonedClock[F[_]] {
+  def now(): F[ZonedDateTime]
```

Comment:
Should we remove `()` as its just a value?

Reply:
Also, cannot we losslessly transform `Instant` to `ZonedDateTime`?

Reply:
I changed the name of the method to `currentTime` to avoid confusion with https://github.com/snowplow/scala-forex/blob/0.7.0/src/main/scala/com.snowplowanalytics/forex/Forex.scala#L174 .

I'm not sure I understand the second part of your comment @chuwy .